### PR TITLE
chore: updates for bzlmod launch

### PR DESCRIPTION
**Pull request by alexeagle**
- we are broken on windows, https://github.com/aspect-build/rules_ts/issues/228
- add the snippet so our next release shows how to use from BCR

### CODE REVIEW

**Comment by kormide** [comment]
Few things left to do:
* Move `bcr` folder to root, change to `.bcr`
* Delete bcr github actions scripts
* Set a fixedReleaser in `.bcr/config.yml`

**Comment by alexeagle** [comment]
okay I'll do those here while I'm in these files

File: .bcr/config.yaml [hunk 1]
```diff
@@ -0,0 +1 @@
+fixedReleaser: alexeagle
```
Comment (kormide) [inline .bcr/config.yaml:1]: The schema for this changed. `fixedReleaser` now takes `login` and `email` fields.

Comment (kormide) [reply .bcr/config.yaml:1]: Also, I think the `.yml` extension is assumed. I should make that more flexible..

**Comment by kormide** [comment]
Just need to rename config.yaml -> config.yml.

**Review by kormide — APPROVED** [review]

**Pull request merged**